test/tracking-index-bounds-test.ts

```diff
@@ -128,6 +128,66 @@ describe('total tracking index bounds', function () {
 });
 
 
-describe('user tracking index bounds', function () {
+describe('user tracking index bounds', async () => {
   // XXX test if small supply/borrow causes users to not accrue rewards
+
+  it('small supply causes user to not accrue rewards', async () => {
+    const {
+      comet, tokens, users: [alice]
+    } = await makeProtocol({
+      base: 'USDC',
+      trackingIndexScale: 1e15,
+      baseTrackingSupplySpeed: 1e8, // supplySpeed=0.0000001 (1e-7) Comp/s
+    });
+    const { USDC } = tokens;
+
+    // allocate and approve transfers
+    await USDC.allocateTo(alice.address, 1e6);
```

Comment:
This actually doesn't accrue rewards because `totalSupplyBase < baseMinForRewards`. We should set `totalSupplyBase > baseMinForRewards` in this test to see the effects of a small supply/borrow when rewards are accruing.

Reply:
Indeed